src/screens/home/HomeScreen.tsx

```diff
@@ -90,36 +70,77 @@ const Content = () => {
         case SystemStatus.BluetoothOff:
           return <BluetoothDisabledView />;
         case SystemStatus.Active:
-        case SystemStatus.Unknown:
           return <NoExposureView />;
+        default:
+          return null;
       }
   }
 };
 
+const CollapsedContent = () => {
+  const [systemStatus] = useSystemStatus();
+  const [notificationStatus, turnNotificationsOn] = useNotificationPermissionStatus();
+  const showNotificationWarning = notificationStatus === 'denied';
+
+  if (systemStatus === SystemStatus.Unknown) {
+    return null;
+  }
+
+  return (
+    <CollapsedOverlayView
+      status={systemStatus}
+      notificationWarning={showNotificationWarning}
+      turnNotificationsOn={turnNotificationsOn}
+    />
+  );
+};
+
+const BottomSheetContent = () => {
+  const [systemStatus] = useSystemStatus();
+  const [notificationStatus, turnNotificationsOn] = useNotificationPermissionStatus();
+  const showNotificationWarning = notificationStatus !== 'granted';
+  const maxWidth = useMaxContentWidth();
+
+  if (systemStatus === SystemStatus.Unknown) {
+    return null;
+  }
+
+  return (
+    <OverlayView
+      status={systemStatus}
+      notificationWarning={showNotificationWarning}
+      turnNotificationsOn={turnNotificationsOn}
+      maxWidth={maxWidth}
+    />
+  );
+};
+
 export const HomeScreen = () => {
   const navigation = useNavigation();
-  React.useEffect(() => {
+  useEffect(() => {
     if (__DEV__) {
       DevSettings.addMenuItem('Show Test Menu', () => {
         navigation.dispatch(DrawerActions.openDrawer());
       });
     }
   }, [navigation]);
 
-  const [systemStatus] = useSystemStatus();
-  const [notificationStatus, turnNotificationsOn] = useNotificationPermissionStatus();
-  const showNotificationWarning = notificationStatus === 'denied';
-  const collapsedContent = useMemo(
-    () => (
-      <CollapsedOverlayView
-        status={systemStatus}
-        notificationWarning={showNotificationWarning}
-        turnNotificationsOn={turnNotificationsOn}
-      />
-    ),
-    [showNotificationWarning, systemStatus, turnNotificationsOn],
-  );
+  // This only initiate system status updater.
+  // The actual updates will be delivered in useSystemStatus().
+  const subscribeToStatusUpdates = useExposureNotificationSystemStatusAutomaticUpdater();
+  useEffect(() => {
+    return subscribeToStatusUpdates();
+  }, [subscribeToStatusUpdates]);
+
+  const startExposureNotificationService = useStartExposureNotificationService();
+  useEffect(() => {
+    startExposureNotificationService();
```

Comment:
I think we should remove this call. We already hook it in the button to turn on EN. Calling start when app opens is annoying when user has EN OFF: the app is opened and shows consent dialog immediately.